js/test/mocha.opts

```diff
@@ -1,3 +1,3 @@
 -r ./test/babel
 -r ./test/mocha.config
--r ignore-styles
+-r ./test/ignoreImports
```

Comment:
So ignoreImports is ignore-styles + ignore ejs, correct?

Reply:
Yes. We also do funny things with markdown (spotted it in my other PR, so adding that as well - all the WebPack loader stuff)